mlir_graphblas/src/lib/GraphBLAS/GraphBLASLowerPass.cpp

```diff
@@ -354,9 +355,17 @@ class LowerCastRewrite : public OpRewritePattern<graphblas::CastOp> {
     }
 
     RankedTensorType inputTensorType = inputType.cast<RankedTensorType>();
+    // sparse_tensor::SparseTensorEncodingAttr inputSparseEncoding =
+    //    sparse_tensor::getSparseTensorEncoding(inputTensorType);
+    // unsigned inputPtrBitWidth = inputSparseEncoding.getPointerBitWidth();
+    // unsigned inputIdxBitWidth = inputSparseEncoding.getIndexBitWidth();
     Type inputValueType = inputTensorType.getElementType();
 
     RankedTensorType outputTensorType = outputType.cast<RankedTensorType>();
+    // sparse_tensor::SparseTensorEncodingAttr outputSparseEncoding =
+    //    sparse_tensor::getSparseTensorEncoding(outputTensorType);
+    // unsigned outputPtrBitWidth = outputSparseEncoding.getPointerBitWidth();
+    // unsigned outputIdxBitWidth = outputSparseEncoding.getIndexBitWidth();
```

Comment:
It's a reminder that we need to deal with bit widths at some point.

Currently, we have many places in the code which assume 64-bit for pointers and indices, so allowing conversion to another bit width would break our code. But someday we will want to tackle that and these will be uncommented.